include/sc2api/sc2_typeenums.h

```diff
@@ -368,7 +396,6 @@ namespace sc2 {
         CANCEL_ADEPTSHADEPHASESHIFT = 2596,  // Target: None.
         CANCEL_BARRACKSADDON = 451,   // Target: None.
         CANCEL_BUILDINPROGRESS = 314,   // Target: None.
-        CANCEL_CREEPTUMOR = 1763,  // Target: None.
```

Comment:
CANCEL_CREEPTUMOR and EFFECT_INFESTEDTERRANS were removed. Is it ok? I believe we should keep it fr older versions.

Reply:
We can if you like, it could cause some confusion on newer versions, but I think there might be a few around like that anyway.  
The other option is to separate the branches for compatibility versions, which might be a good option overall, as some of the changes might be problematic on older versions, but I'll go with whatever you think is best :)

Reply:
I would follow the branches approach, but it'll lead to problems with replay analysers as one could use older replays with the newest API. Because of it, I suggest to keep old IDs as is.

Reply:
cool, I'll update the PR tonight